File: Angular/src/app/add-module/interest-dialog/domain/section.ts
import { Chicklets } from './chicklets';
export class InterestsSection {
    constructor(private sectionId: string,
        private userId: string,
        private operationType: string,
        private chicklets: Chicklets[]) {
        this.sectionId = sectionId;
        this.userId = userId;
        this.operationType = operationType;
        this.chicklets = chicklets;
    }
}

File: ProjectService/src/main/java/com/stackroute/projectservice/domain/Section.java
package com.stackroute.projectservice.domain;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

/* section class is the input format for this service*/
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Section {
    private String sectionId;
    private String userId;
    private String operationType;
    private Chicklets[] chicklets;
}
